isso/__init__.py

```diff
@@ -295,11 +295,14 @@ def main():
         except ImportError:
             run_simple(host, port, make_app(conf), threaded=True,
                        use_reloader=conf.getboolean('server', 'reload'))
-    else:
+    elif conf.get("server", "listen").startswith("unix://"):
         sock = conf.get("server", "listen").partition("unix://")[2]
         try:
             os.unlink(sock)
         except OSError as ex:
             if ex.errno != errno.ENOENT:
                 raise
         wsgi.SocketHTTPServer(sock, make_app(conf)).serve_forever()
+    else:
+        logger.error("server.listen must specif a protocol of http:// or unix://")
```

Comment:
nit: should say "specify" instead of "specif"

Reply:
Fixed